dm/dm/worker/source_worker_test.go

```diff
@@ -818,7 +818,26 @@ func (t *testServer) TestQueryValidator(c *C) {
 	ret, err = w.GetValidatorTableStatus("testQueryValidator", pb.Stage_InvalidStage)
 	c.Assert(err, IsNil)
 	c.Assert(len(ret), Equals, 2)
-	// the return order is not stable
+	if ret[0].SrcTable == expected[0].SrcTable {
```

Comment:
maybe simplified this ut too. so we can remove all failpoint `TestQueryValidator`, and can do ut directly.

Reply:
2e7e4b5